pkg/skaffold/instrumentation/meter.go

```diff
@@ -110,20 +110,20 @@ func init() {
 	}()
 }
 
-func InitMeterFromConfig(config *latest.SkaffoldConfig) {
-	meter.PlatformType = yamltags.GetYamlTag(config.Build.BuildType)
-	for _, artifact := range config.Pipeline.Build.Artifacts {
-		if _, ok := meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)]; ok {
-			meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)]++
-		} else {
-			meter.Builders[yamltags.GetYamlTag(artifact.ArtifactType)] = 1
-		}
-		if artifact.Sync != nil {
-			meter.SyncType[yamltags.GetYamlTag(artifact.Sync)] = true
+func InitMeterFromConfig(configs []*latest.SkaffoldConfig) {
+	meter.PlatformType = yamltags.GetYamlTag(configs[0].Build.BuildType) // TODO: support multiple build types
```

Comment:
@IsaacPD can we modify `PlatformType` to be a slice instead?

Reply:
yeah that should be fine